testutils/helper/install.go

```diff
@@ -82,6 +81,16 @@ func NewSoloTestHelper(configFunc TestConfigFunc) (*SoloTestHelper, error) {
 		testConfig = configFunc(defaults)
 	}
 
+	if testConfig.GlooctlExecName == "" {
+		if goarch, exists := os.LookupEnv("GOARCH"); exists && (goarch == "amd64" || goarch == "arm64") {
+			testConfig.GlooctlExecName = "glooctl-" + runtime.GOOS + "-" + goarch
+		} else if runtime.GOARCH == "arm64" {
```

Comment:
nit: I thought that per parking lot this was no longer needed?

Reply:
It appeared that way at first but thats because we were using `go test` which sets the GOARCH, if we use something like `ginkgo` then this  doesnt get set and we hit the case on 87.

Reply:
Can we add comment to this affect?